atd-vzv/src/views/summary/FatalitiesByTimeOfDay.js

```diff
@@ -129,85 +130,95 @@ const FatalitiesByTimeOfDayWeek = () => {
   return (
     <Container>
       <Row>
-        <h3>Traffic fatalities by time and day</h3>
+        <Col md="2"></Col>
+        <Col md="8">
+          <Nav tabs>
+            <NavItem>
+              <NavLink
+                className={classnames({ active: activeTab === "6" })}
+                onClick={() => {
```

Comment:
These two functions called in these onClick handlers could be combined into one function that takes two arguments if  you wanted. Maybe something like: 

```
const handleYearTabClick = (tabId, dataView) = {
   toggle(tabId);
   setDataView(dataView);
}
```
If you start the tab labeling at "0", you could also have the tab and dataView arguments be the same number since the number represents the age (# of years ago) of the data shown in each tab.

Reply:
Good call! Once I switched the labeling to match, I only needed one function, so I got rid of the dataView variable and function in state.